### DISCUSSION TITLE
How can i request account ? Always display 404 not found

### DISCUSSION THREAD

**Discussion by Hi1805** [microsoft/PlanetaryComputer#395] [Q&A] [answered]
When i clicked "request account" in header.
My screen will display ?
![image](https://github.com/user-attachments/assets/ef01425d-5f19-4fd6-882f-6cd9fe3ca8e0)

Could you please help me how i can help account for pc_key ?

**Comment by 777arc** [comment]
What page are you referring to that has a "request account" in the header?

**Reply by Hi1805** [reply]
@777arc  Thank you for reply, Page i access here:
![image](https://github.com/user-attachments/assets/5e794392-ee38-4f61-ab5d-4998673e03c7)

https://planetarycomputer.developer.azure-api.net/#

I click always show 404 not found

**Reply by 777arc** [reply]
Ah thank you, that is an old page, you no longer need an account or API key to access the Planetary Computer.  Is there something specific you are trying to do?

**Reply by Hi1805** [reply]
Hi @777arc, 
I am researching about farmvibes AI. In workflow.
I need to `pc_key`, I Guess pc_key equal api key
![image](https://github.com/user-attachments/assets/0a1fe3f8-19e0-40aa-a6b9-3b8154e36b15) 

I am trying find pc_key :(((((

Could you please help me ?

**Reply by 777arc** [reply, marked as answer]
You no longer need pc_key, so leave it empty
